node/pkg/db/governor.go

```diff
@@ -112,10 +112,60 @@ func UnmarshalTransfer(data []byte) (*Transfer, error) {
 	return t, nil
 }
 
+type PendingTransfer struct {
+	ReleaseTime time.Time
+	Msg         common.MessagePublication
+}
+
+func (p *PendingTransfer) Marshal() ([]byte, error) {
+	buf := new(bytes.Buffer)
+
+	vaa.MustWrite(buf, binary.BigEndian, uint32(p.ReleaseTime.Unix()))
+
+	b, err := p.Msg.Marshal()
+	if err != nil {
+		return buf.Bytes(), fmt.Errorf("failed to marshal pending transfer: %w", err)
+	}
+
+	vaa.MustWrite(buf, binary.BigEndian, b)
+
+	return buf.Bytes(), nil
+}
+
+func UnmarshalPendingTransfer(data []byte) (*PendingTransfer, error) {
+	p := &PendingTransfer{}
+
+	reader := bytes.NewReader(data[:])
+
+	unixSeconds := uint32(0)
+	if err := binary.Read(reader, binary.BigEndian, &unixSeconds); err != nil {
+		return nil, fmt.Errorf("failed to read pending transfer release time: %w", err)
+	}
+
+	p.ReleaseTime = time.Unix(int64(unixSeconds), 0)
+
+	buf := make([]byte, reader.Len())
+	n, err := reader.Read(buf)
+	if err != nil || n == 0 {
+		return nil, fmt.Errorf("failed to read pending transfer msg [%d]: %w", n, err)
+	}
+
+	msg, err := common.UnmarshalMessagePublication(buf)
+	if err != nil {
+		return nil, fmt.Errorf("failed to unmarshal pending transfer msg: %w", err)
+	}
+
+	p.Msg = *msg
+	return p, nil
+}
+
 const transfer = "GOV:XFER:"
 const transferLen = len(transfer)
 
-const pending = "GOV:PENDING:"
+const oldPending = "GOV:PENDING:"
+const oldPendingLen = len(oldPending)
+
+const pending = "GOV:PENDING2:"
```

Comment:
Can we add some comments to help understand the nuances between GOV:PENDING and GOV:PENDING2?

Reply:
I believe the intent is that this is migration process, such that we will have DB entries with GOV:PENDING and GOV:PENDING2, but they will be converted to GOV:PENDING2's.

Reply:
Added a comment.